kythe-verification/testdata/basic/ImportsRef.hs

```diff
@@ -0,0 +1,8 @@
+-- Verify the references to the imported modules
```

Comment:
The current way won't actually crosslink I fear, since when we emit a module entry, the package is part of its signature (see `ModuleTick`). See comment above. After fixing, please add a VName verification on `DS` to check that the package is also included. See `Module.hs` in the verifier testdata on how to do VName asserts.